fabfile.py

```diff
@@ -13,7 +13,8 @@
     docker_execute_command,
     generate_satellite_docker_clients_on_rhevm,
     refresh_subscriptions_on_docker_clients,
-    remove_all_docker_containers
+    remove_all_docker_containers,
+    docker_cleanup_containers
```

Comment:
docker_cleanup_containers will delete the boxes which are older than a day , and after the job is created , will remove all instances of remove_all_docker_containers, will raise a issue to remove them once job is created and PR is merged in robotello-ci.